src/utils/sources-tree/utils.js

```diff
@@ -34,7 +34,7 @@ export function isInvalidUrl(url: Object, source: Object) {
   return (
     IGNORED_URLS.indexOf(url) != -1 ||
     !source.get("url") ||
-    source.get("loading") ||
+    source.loadedState === "loading" ||
```

Comment:
Ah, i see. in this case it should be source.get("loadedState") !== "loading". so if it is unloaded it is fine. Sorry -- i misunderstood the case earlier